benchmarks/DASB/utils/tokens.py

```diff
@@ -0,0 +1,382 @@
+"""
+Unified interface for token extraction and pretrained embeddings handling for speech tokenizers.
+
+Authors
+---------
+* Jarod Duret, 2024
+"""
+
+import math
+import logging
+import pathlib as pl
+import kaldiio
```

Comment:
What are the advantages of using `kaldiio` format over, for example a text file with list of utterance ids and a folder of `.npz` archives, one for each utterance?